test/org.jkiss.dbeaver.ext.generic.test/OSGI-INF/l10n/bundle_ro_RO.properties

```diff
@@ -0,0 +1 @@
+#Generated by ResourceBundle Editor (http://essiembre.github.io/eclipse-rbe/)
```

Comment:
Please remove empty file.

Reply:
Solved in [dc41b1b](https://github.com/v613/dbeaver/commit/dc41b1b8db11db37af504a75957881ef7145b9a8)